bazel/deps.bzl

```diff
@@ -266,8 +266,15 @@ def stratum_deps():
             # TODO(max): This is kind of hacky and should be improved.
             # Each string is a new bash shell, use && to run dependant commands.
             patch_cmds = [
-                "wget -qO- https://github.com/opennetworkinglab/OpenNetworkLinux/releases/download/onlpv2-dev-1.0.1/linux-4.14.49-OpenNetworkLinux.tar.xz | tar xz",
-                "export CC=gcc CXX=g++ CFLAGS='-Wno-error=unused-result -fno-pie' KERNDIR=$(realpath ./linux-4.14.49-OpenNetworkLinux) && cd src/gpl-modules/systems/linux/user/x86-smp_generic_64-2_6 && make clean -j && make",
+                "wget --quiet -O linux-headers-4.19.0-12-2-common.deb 'https://sonic-build.azurewebsites.net/api/sonic/artifacts?branchName=202106&platform=barefoot&buildId=116439&target=target%2Fdebs%2Fbuster%2Flinux-headers-4.19.0-12-2-common_4.19.152-1_all.deb'",
+                "wget --quiet -O linux-headers-4.19.0-12-2-amd64.deb 'https://sonic-build.azurewebsites.net/api/sonic/artifacts?branchName=202106&platform=barefoot&buildId=116439&target=target%2Fdebs%2Fbuster%2Flinux-headers-4.19.0-12-2-amd64_4.19.152-1_amd64.deb'",
+                "sudo apt-get install -y --no-install-recommends ./linux-headers-4.19.0-12-2-common.deb",
+                "sudo apt-get install -y --no-install-recommends ./linux-headers-4.19.0-12-2-amd64.deb",
```

Comment:
Running `apt-get` and modifying global state in a Bazel rule is a little weird. Maybe just unpack locally instead?

Reply:
Maybe:

```sh
wget --quiet -O linux-headers-4.19.0-12-2-common.deb 'https://sonic-build.azurewebsites.net/api/sonic/artifacts?branchName=202106&platform=barefoot&buildId=116439&target=target%2Fdebs%2Fbuster%2Flinux-headers-4.19.0-12-2-common_4.19.152-1_all.deb'
wget --quiet -O linux-headers-4.19.0-12-2-amd64.deb 'https://sonic-build.azurewebsites.net/api/sonic/artifacts?branchName=202106&platform=barefoot&buildId=116439&target=target%2Fdebs%2Fbuster%2Flinux-headers-4.19.0-12-2-amd64_4.19.152-1_amd64.deb'
dpkg -x linux-headers-4.19.0-12-2-amd64.deb linux-headers-4.19.0-12-2-amd64
dpkg -x linux-headers-4.19.0-12-2-common.deb linux-headers-4.19.0-12-2-common
mkdir /tmp/linux-headers-4.19.0-12-2-merged
rsync -ahPL linux-headers-4.19.0-12-2-common /tmp/linux-headers-4.19.0-12-2-merged
rsync -ahPL linux-headers-4.19.0-12-2-amd64 /tmp/linux-headers-4.19.0-12-2-merged

# Build
export CC=gcc CXX=g++ CFLAGS='-Wno-error=unused-result -fno-pie' KERNDIR=/tmp/linux-headers-4.19.0-12-2-merged && cd src/gpl-modules/systems/linux/user/x86-smp_generic_64-2_6 && make clean -j && make

# Cleanup
rm -rf ./linux-headers-4.19.0-12-2-common.deb ./linux-headers-4.19.0-12-2-amd64.deb linux-headers-4.19.0-12-2-common linux-headers-4.19.0-12-2-amd64 /tmp/linux-headers-4.19.0-12-2-merged
```

Reply:
NVM, I think the `apt` way is simpler since we need to fix the symlink(tools and scripts that linked to linux-kbuild package)

Reply:
@pudelkoM can you also help us upload those deb to the `stratum/sonic-base-image` repo?(and make it public)
So we don't need to worry if they delete those artifacts

Reply:
Sure, will upload them and then update the links here.

About the `apt` part: Thanks for checking out the manual extraction way. Symlinks are indeed a problem. If the current approach does not break anything, I'm ok with using `apt-get`.

Reply:
You could be right with the cleanup part. Just run `apt remove`? And `rm -rf` the other directory?

Reply:
Right, I think that should work

Reply:
Links are now public and the debs are uninstalled after build.